### DISCUSSION TITLE
Segment.io integration

### DISCUSSION THREAD

**Discussion by timuric** [saleor/apps#19] [Integrations & features]
[Segment](https://segment.com/) allows users to stream analytics events to a variety of sources like Google Analytics, Mixpanel which would provide enriched ecommerce data without need for client side tracking.

Potential events to track:

- Orders created
- Refunds
- Fulfillments
- Account creation
- Cart creation
- Cart abandonment
- Voucher redemption

**Comment by peteror** [comment]
Take a look at how to add [Rudderstack](https://www.rudderstack.com/) as well - very similar functionality, but has an open-source version too. The APIs are compatible with Segment, so it may be an easy addition.
